targets/ingester/src/transform/agreements/kaliArticleBytheme.ts

```diff
@@ -1,18 +1,30 @@
+import { KaliArticleHDN } from "@shared/types";
+import {
+  Agreement,
+  AgreementArticle,
+  AgreementArticleData,
+  AgreementArticleWithParentSections,
+  AgreementSection,
+} from "@socialgouv/kali-data-types";
```

Comment:
Prefer `import type`
```suggestion
import type {
  Agreement,
  AgreementArticle,
  AgreementArticleData,
  AgreementArticleWithParentSections,
  AgreementSection,
} from "@socialgouv/kali-data-types";
```